backend/test/institution_handlers_tests/institution_collection_handler_test.py

```diff
@@ -39,11 +39,28 @@ def test_get(self, verify_token):
         # Call the get method
         all_institutions = self.testapp.get("/api/institutions?page=0&&limit=2").json
 
-        self.assertEqual(len(all_institutions), 2,
+        self.assertEqual(len(all_institutions['institutions']), 2,
                          "The length of all institutions list should be 2")
 
         self.assertEqual(all_institutions['institutions'][0]['name'], first_inst.name,
                          "The name of institituion should be equal to the first_inst name")
 
         self.assertEqual(all_institutions['institutions'][1]['name'], third_inst.name,
-                         "The name of institution should be equal to the third_inst name")
\ No newline at end of file
+                         "The name of institution should be equal to the third_inst name")
+
+    @patch('util.login_service.verify_token', return_value={'email': 'user@example.com'})
+    def test_get_all_institutions(self, verify_token):
+        """Test the get method."""
+        # create models
+        # new User
+        user = mocks.create_user('user@example.com')
+        
+        for i in xrange(15):
+            mocks.create_institution()
```

Comment:
I don't understand the point of using this for loop

Reply:
I need more than 10 institutions created in the database to get them all. I've modified the handler to accept the option of get all institutions without pagination. The default limit value is 10